src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -85,8 +85,15 @@ def update_data(self):
             if not ent:
                 continue
 
+            bridge_port_id_attr = ""
+            try:
+                bridge_port_id_attr = ent["SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID"]
+            except KeyError as e:
+                mibs.logger.error("SyncD 'ASIC_DB' includes invalid FDB_ENTRY '{}': failed to get bridge_port_id, exception: {}".format(fdb_str, e))
+                continue
```

Comment:
>continue [](http://example.com/codeflow?start=16&length=8)

If we continue and it will the error message will repeat. In a real device, how frequent is the same error messages? do we need to damp the speed?

Reply:
The log will repeat every 5 seconds, seems not necessary to add more delay:

in https://github.com/sonic-net/sonic-snmpagent/blob/master/src/sonic_ax_impl/main.py

# Background task update frequency ( in seconds )
DEFAULT_UPDATE_FREQUENCY = 5

Reply:
Fixed, every FDB only output error log once.

Reply:
Thanks for the fix! Further suggestion: flush `broken_fdbs` in `reinit_data()` in order to prevent it increasing size forever in case the FDB entries become crazy.

Reply:
Fixed